Simplified/NYPLTenPrintCoverView+NYPLImageAdditions.h

```diff
@@ -2,7 +2,10 @@
 
 @interface NYPLTenPrintCoverView (NYPLImageAdditions)
 
++ (UIImage *)detailImageForBook:(NYPLBook *const)book;
++ (UIImage *)thumbnailImageForBook:(NYPLBook *const)book;
+
 // Must be called on the main thread.
-+ (UIImage *)imageForBook:(NYPLBook *const)book;
++ (UIImage *)imageForBook:(NYPLBook *const)book withSize:(CGSize)size;
```

Comment:
The comment "must be called..." also applies to the two methods you added, so go ahead and move the comment atop all three.